aws-predictions/src/test/resources/configuration-with-interpret.json

```diff
@@ -0,0 +1,10 @@
+{
+  "defaultRegion": "us-east-1",
+  "interpret": {
+    "interpretText": {
+      "region": "us-west-2",
+      "type": "SENTIMENT",
```

Comment:
This is based off of the actual `amplifyconfiguration.json` file that was generated by CLI. I will add support for lower case also.

Reply:
on second thought, I think it is unnecessary, since we don't want to encourage them to directly edit these anyways. The best way to edit these without error would be to update via: 
```
amplify update predictions
amplify push
```